decayedlog/kvmigrations/migration1/migration.go

```diff
@@ -0,0 +1,42 @@
+package migration1
```

Comment:
Why do we need a new package for this vs just putting it where the other optional migrations are?

Reply:
I tried both ways but chose this one because:

1. I added a new data format to the ReplaySet so we need to make sure there is no old data in the bucket to be read otherwise the decoding does not work.

2. I went with an own package because if I have to touch the mandatory migration in the channeldb package I need to change the func signature because the sphinxdb is a different db than the channel.db so I changing the function signature for every migration already happened was imo not super lean. 

In any ways open to move the migration into the channeldb package if people think that is the better choice.

Reply:
Would also prefer to keep the migration in one place, which is the more correct way - imagine if we have a migration that touches multiple tables, then we won't be able to make it if it's living in the subpackages. Unless we are moving decalylog into its own db, the migration should live in the same place as they are all under channel db.